### PR TITLE
Feature/graph

### PR DESCRIPTION
**Pull request by akinolu52**
## What
*Added root node to the graph view*
*Differentiate between active, other and source nodes*

### CODE REVIEW

File: re_data_ui/src/pages/GraphView.tsx [hunk 1]
```diff
@@ -1,28 +1,29 @@
+/* eslint-disable comma-dangle */
```
Comment (dejii) [inline re_data_ui/src/pages/GraphView.tsx:1]: 👀

File: re_data_ui/src/pages/GraphView.tsx [hunk 2]
```diff
@@ -51,33 +64,40 @@ const generateGraph = (overview: OverviewData) => {
...
+    if (supportedResourcesTypes.includes(details.resource_type) && details.package_name !== 're_data') {
       const modelId = `${details.database}.${details.schema}.${details.name}`.toLowerCase();
       const node: VisNode = {
         id: modelId,
-        label: modelName,
+        label: details.name,
         shape: 'box',
+        color: {
+          background: resourceTypeColors[details.resource_type],
+        }
```
Comment (dejii) [inline re_data_ui/src/pages/GraphView.tsx:79]: At this array length, doing `Array.includes()` might not be an issue but I think we should go with using a set so we have constant lookups.

File: re_data_ui/src/pages/GraphView.tsx (hunk 1 above)
Comment (akinolu52) [reply re_data_ui/src/pages/GraphView.tsx:1]: Lol 😂😂

Oh yes I'll remove it ☺️

File: re_data_ui/src/pages/GraphView.tsx (hunk 2 above)
Comment (akinolu52) [reply re_data_ui/src/pages/GraphView.tsx]: done, thanks

**Review by dejii — APPROVED** [review]

**Pull request merged**